docs/source/delta-storage.md

```diff
@@ -66,11 +66,10 @@ In this default mode, <Delta> supports concurrent reads from multiple clusters,
 
 This section explains how to quickly start reading and writing Delta tables on S3 using single-cluster mode. For a detailed explanation of the configuration, see [_](#setup-configuration-s3-multi-cluster).
 
-#. Use the following command to launch a Spark shell with <Delta> and S3 support (assuming you use Spark 3.5.0 which is pre-built for Hadoop 3.3.4):
-
    ```bash
    bin/spark-shell \
-    --packages io.delta:delta-spark_2.12:3.1.0,org.apache.hadoop:hadoop-aws:3.3.4 \
+    --packages io.delta:delta-spark_2.12:3.0.0,org.apache.hadoop:hadoop-aws:3.3.4 \
```

Comment:
why this change?

Reply:
I opened this PR 2 months ago when the version was 3.0.0, anyway, I updated it to 3.1.0.